### PR TITLE
Feat/item listings

### PR DESCRIPTION
**Pull request by siddhantk232**

### CODE REVIEW

File: src/apps/inventory/views/Listings/Item/index.js [hunk 1]
```diff
@@ -102,3 +212,17 @@ export default function ItemListing() {
          </StyledWrapper>
       )
 }
+
+const CellColumnContainer = styled.div`
```
Comment (thesanjeevsharma) [inline src/apps/inventory/views/Listings/Item/index.js:216]: Put styles in a different file.

File: src/apps/inventory/views/Listings/Item/index.js [hunk 2]
```diff
@@ -84,15 +92,117 @@ export default function ItemListing() {
...
+                                    <div style={{ padding: '5px' }}>
+                                       {processing.committed} {processing.unit}
+                                    </div>
+                                 ))}
+                              </CellColumnContainer>
+                           </TableCell>
+                           <TableCell>
+                              <CellColumnContainer>
+                                 {item.bulkItems?.map(processing => (
+                                    <div style={{ padding: '5px' }}>
```
Comment (thesanjeevsharma) [inline src/apps/inventory/views/Listings/Item/index.js:183]: Avoid styling like this.

File: src/apps/inventory/views/Listings/Item/index.js (hunk 1 above)
Comment (siddhantk232) [reply src/apps/inventory/views/Listings/Item/index.js:216]: these styled components are only meant to be used in this listing view. so there is no point in moving them into different files

File: src/apps/inventory/views/Listings/Item/index.js (hunk 2 above)
Comment (siddhantk232) [reply src/apps/inventory/views/Listings/Item/index.js:183]: you mean those inline styles?

File: src/apps/inventory/views/Listings/Item/index.js (hunk 1 above)
Comment (thesanjeevsharma) [reply src/apps/inventory/views/Listings/Item/index.js:216]: Structure matters.

File: src/apps/inventory/views/Listings/Item/index.js (hunk 2 above)
Comment (thesanjeevsharma) [reply src/apps/inventory/views/Listings/Item/index.js:183]: Yeah

File: src/apps/inventory/views/Listings/Item/index.js (hunk 1 above)
Comment (siddhantk232) [reply src/apps/inventory/views/Listings/Item/index.js]: done :)

**Pull request merged**